server/container_create_linux.go

```diff
@@ -591,6 +591,18 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		})
 	}
 
+	if sb.ContainerEnvPath() != "" {
+		if err := sb.SetContainerEnvFile(); err != nil {
+			return nil, err
+		}
```

Comment:
this should be called in `server/sandbox_run_linux.go` as it's recreating containerenv file each time a container is created, when i believe the intention is to share the container env file for the whole pod